app/views/messages/_form.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="border border-gray-300 sm:rounded-lg shadow-sm overflow-hidden focus-within:border-gray-500 focus-within:ring-1 focus-within:ring-gray-500">
   <%= form.label :body, t(".help"), class: "sr-only" %>
-  <%= form.text_area :body, rows: 4, placeholder: t(".placeholder"), class: "block w-full py-3 border-0 sm:rounded-lg sm:rounded-b-none resize-y focus:ring-0 sm:text-sm" %>
+  <%= form.text_area :body, rows: 4, placeholder: t(".placeholder"), class: "block w-full py-3 border-0 sm:rounded-lg sm:rounded-b-none resize-y focus:ring-0 sm:text-sm", data: {action: "keydown->form#cmdEnterSubmit"} %>
```

Comment:
```suggestion
  <%= form.text_area :body, rows: 4, placeholder: t(".placeholder"), data: {action: "keydown->form#cmdEnterSubmit"}, class: "block w-full py-3 border-0 sm:rounded-lg sm:rounded-b-none resize-y focus:ring-0 sm:text-sm" %>
```

Just because `class` can grow so I like to leave it at the end.